frontend/web/components/App.js

```diff
@@ -183,7 +192,15 @@ const App = class extends Component {
             )
             return
           }
-
+          if (
```

Comment:
I could be being silly but is this duplicated from 170 ?

Reply:
It was part of an old logic, I've updated it and removed it.